multicall/call.py

```diff
@@ -22,7 +22,7 @@ def __init__(
         self, 
         target: AnyAddress, 
         function: Union[str,Iterable[Union[str,Any]]], # 'funcName(dtype)(dtype)' or ['funcName(dtype)(dtype)', input0, input1, ...]
-        returns: Optional[Iterable[Tuple[str,Callable]]] = None, 
+        returns: Optional[Iterable[Tuple[str,Optional[Callable]]]] = None,
```

Comment:
Would it also be useful to change that `Tuple` typehint to another `Iterable`? Examples all used to be nested lists until #55 where it simply updated the Readme to match the typing but there's no reason to limit it.

Changing to `Iterable` would seemingly cover all use cases -- when developing, typing `[[` is much easier than switching out to tuples.